docs/contracts.rst

```diff
@@ -765,9 +772,9 @@ Frontier and Homestead, but this might change with Serenity). Log and
 event data is not accessible from within contracts (not even from
 the contract that created them).
 
-SPV proofs for logs are possible, so if an external entity supplies
+"Simple payment verification" proofs for logs are possible, so if an external entity supplies
```

Comment:
I'd add (SPV) here after the definition purely because I have seen people use the term online a lot.